content/vault/v1.21.x (rc)/content/docs/sysadmin/snapshots/recover-a-secret/replicated-cluster.mdx

```diff
@@ -36,6 +36,37 @@ Load the snapshot holding the secret you want to recover:
 
 <Tabs>
 
+<Tab heading="Web UI" group="ui">
+
+1. Open a web browser to access the Vault UI and sign in to the root namespace.
+   <Tip title="Namespace restriction">
+
+   Snapshot load and unload operations are restricted to the root namespace. All other snapshot operations 
+   can be performed in other namespaces. 
+
+    </Tip>
+
+    <Tip title="Permissions Required">
+        This requires snapshot management permissions.
+    </Tip>
```

Comment:
```suggestion

Snapshot load and unload operations are restricted to the root namespace. All other snapshot operations 
can be performed in other namespaces. 

1. Open the Vault GUI and sign in to the root namespace. While you can perform
   other snapshot operations from any namespace, you must perform load and
   unload operations under the `root` namespace.
```
Style correction: write in active voice
Structural correction: whenever possible, include information in normal text rather than asides

Since the permission requirement is a prerequisite, we should add it to the "Before you start" section, especially since it's probably not just a GUI requirement